features/step_definitions/purap/requisition.rb

```diff
@@ -211,3 +217,10 @@
     (req_page.requisition_status).should == status_desired
   end
 end
+
+
+And /^I capture the Requisition number$/ do
+  on page_class_for('Requisition') do |page|
+    @requisition_id = page.is_requisition_document ? page.requisition_id : nil
```

Comment:
Are you sure you should return nil? If the step is supposed to capture the number maybe it should fail if it can't find it.